Description: Stop trying to fiddle with non-existent kernels when
  we explicitely didn't install any via --linux-packages=none
Author: Adam Conrad <adconrad@ubuntu.com>

--- live-build-3.0~a24.orig/scripts/build/lb_binary_linux-image
+++ live-build-3.0~a24/scripts/build/lb_binary_linux-image
@@ -24,7 +24,7 @@ Arguments "${@}"
 Read_conffiles config/all config/common config/bootstrap config/chroot config/binary config/source
 Set_defaults
 
-if [ "${LB_BINARY_IMAGES}" = "virtual-hdd" ]
+if [ "${LB_BINARY_IMAGES}" = "virtual-hdd" ] || [ "${LB_LINUX_PACKAGES}" = "none" ]
 then
 	exit 0
 fi
